py/server/deephaven/table_listener.py

```diff
@@ -355,20 +330,32 @@ def __init__(self, t: Table, listener: Union[Callable, TableListener], descripti
             listener (Union[Callable, TableListener]): listener for table changes
             description (str, optional): description for the UpdatePerformanceTracker to append to the listener's entry
                 description, default is None
+            dependencies (Union[Table, Sequence[Table]]): tables that must be satisfied before the listener's execution. A
+                refreshing table is considered to be satisfied if all updates to the table have been processed in the current
```

Comment:
```suggestion
                refreshing table is considered to be satisfied if all possible updates to the table have been processed in the current
```

Satisfied means: "this table has been updated" (which happens just once/cycle) _OR_ "this table will not update"